lib/membrane/video_compositor/queue/strategy/live.ex

```diff
@@ -48,22 +49,31 @@ defmodule Membrane.VideoCompositor.Queue.Strategy.Live do
         vc_init_options:
           vc_init_options = %VideoCompositor{
             output_stream_format: %RawVideo{framerate: framerate},
-            queuing_strategy: %Live{latency: latency}
+            queuing_strategy: %Live{latency: latency, eos_strategy: eos_strategy}
           }
       }) do
     {[],
      %State{
        output_framerate: framerate,
        custom_strategy_state: %LiveState{
-         latency: latency
+         latency: latency,
+         eos_strategy: eos_strategy
        },
        handler: HandlerState.new(vc_init_options)
      }}
   end
 
+  @impl true
+  def handle_playing(_ctx, state) do
+    {[stream_format: {:output, %CompositorCoreFormat{pad_formats: %{}}}], state}
+  end
+
   @impl true
   def handle_pad_added(pad, context, state) do
-    state = Bunch.Struct.put_in(state, [:pads_states, pad], PadState.new(context.options))
+    state =
```

Comment:
I would put it in `handle_start_of_stream`

Reply:
We need to have access to input pad options, to pass them into `handler`, which are only available in `handle_pad_added` and `handle_pad_removed`, so we need to "register" the input pad queue here or keep this information somehow. I changed the logic slightly, such that it should handle unlinking pads without sending any buffers.